.github/workflows/check-pull-request.yml

```diff
@@ -37,6 +37,24 @@ jobs:
         with:
           fetch-depth: 1
 
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'adopt'
+          java-version: '17'
+          cache: 'gradle'
+
       - name: Run Tests using Gradle Wrapper
-        run: ./gradlew test --no-daemon
+        run: |
+          ./gradlew --no-daemon test jacocoTestReport -x assemble
+
+      - name: Publish coverage reports
+        uses: codecov/codecov-action@v4
+        with:
+          fail_ci_if_error: true 
+          files: ./bridge/build/reports/jacoco/test/jacocoTestReport.xml
+          name: frontend-library
+          token: ${{ secrets.CODECOV_TOKEN }}
+          verbose: true
+          plugin: noop
 
```

Comment:
Remove the extra empty line you added.

Reply:
> It's not about adding an extra newline at the end of a file, it's about not removing the newline that should be there.
A [text file](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_392), under unix, consists of a series of [lines](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_205), each of which ends with a [newline character](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap03.html#tag_03_238) (\n). A file that is not empty and does not end with a newline is therefore not a text file.

-- [source](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

Reply:
I know it, but you didn't understand what I was saying.
In `check-push.yml` file we have the newline character, which is good and refers to what you mention. But in the other one, `check-pull-request.yml`, we have two newline characters. 
The images of both files are shown below.

`check-push.yml`
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/76bcee50-f122-4dd0-9e9f-8b9a60cd48f2)

`check-pull-request.yml`
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/81e78e58-9127-4902-afa2-119d82f3e169)

Reply:
Are you sure about that? the git diff ony shows one...

`$ git diff --staged --patch origin/main -- .github/workflows/check-pull-request.yml  | tail`

![image](https://github.com/Pending-Name-21/frontend-library/assets/64939916/4c1efb07-253f-46f6-8215-3899641ddf3a)

Reply:
Did you see the difference?
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/e98207c0-931c-4720-a896-54c19475c972)

Reply:
Did you see the difference between both?
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/e9e4b7af-0af8-45bc-bfee-2d0f8b2a8042)

Reply:
Yeah, about that... the context of this PR is just the pull-request yml.

Reply:
I was not talking about the context of the PR, I mentioned that file in order to see extra empty line you added besides of required one.

Reply:
bruh, I'm telling you, there are no extra lines. It has 62 lines, the 62nd line is the empty line

Reply:
Are you sure? Why don't you review it using an editor?

Reply:
No changes:
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/123677cb-94c8-4247-b2ed-f665defac336)

Deleting the extra line you cannot see.
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/ebcd313e-9f4f-414e-b8a8-bc91046914ff)

Reply:
No changes:
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/cc400292-6aa1-4873-9134-52da4c1a3662)

Deleting the extra newline you cannot see.
![image](https://github.com/Pending-Name-21/frontend-library/assets/73414963/3d9897ea-cc0f-4127-bf4e-7e61e57c6c70)

Reply:
I've opened with an editor as well.

can you try using this command?

`$  git show origin/task-150/test-coverage:.github/workflows/check-pull-request.yml | wc -l`

it should print 62, the amount of lines that file has